projects/lzma/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# build and install fuzzers
+make clean
```

Comment:
Are you sure that CFLAGS and CXXFLAGS are propagated properly to the underlying code?

```
$ python infra/helper.py check_build lzma
Running: docker run --rm -i --privileged -e FUZZING_ENGINE=libfuzzer -e SANITIZER=address -v /usr/local/google/home/mmoroz/projects/oss-fuzz/build/out/lzma:/out -t gcr.io/oss-fuzz-base/base-runner test_all
INFO: performing bad build checks for /out/lzmadec_fuzzer.
INFO: performing bad build checks for /out/delta_fuzzer.
INFO: performing bad build checks for /out/lzma2dec_fuzzer.
INFO: performing bad build checks for /out/7zcrc_fuzzer.
Broken fuzz targets (4):
7zcrc_fuzzer:
BAD BUILD: /out/7zcrc_fuzzer seems to have only partial coverage instrumentation.
BAD BUILD: /out/7zcrc_fuzzer does not seem to be compiled with ASan.
delta_fuzzer:
BAD BUILD: /out/delta_fuzzer seems to have only partial coverage instrumentation.
BAD BUILD: /out/delta_fuzzer does not seem to be compiled with ASan.
lzma2dec_fuzzer:
BAD BUILD: /out/lzma2dec_fuzzer does not seem to be compiled with ASan.
lzmadec_fuzzer:
BAD BUILD: /out/lzmadec_fuzzer does not seem to be compiled with ASan.
ERROR: 100% of fuzz targets seem to be broken. See the list above for a detailed information.
Check build failed.
```